web/satellite/src/components/account/billing/depositAndBilling/Countdown.vue

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="countdown-container">
+        <div v-if="isExpired">{{date}}</div>
+        <div class="row" v-else>
+            <p>Expires in </p>
+            <p class="digit margin">{{ minutes | twoDigits }}</p>
+            <p>:</p>
+            <p class="digit">{{ seconds | twoDigits }}</p>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+import { BillingHistoryItemType } from '@/types/payments';
+
+@Component
+export default class Countdown extends Vue {
+    @Prop({default: () => new Date()})
+    private readonly expirationDate: Date;
+    @Prop({default: () => new Date()})
+    private readonly startDate: Date;
+    @Prop({default: 0})
+    private readonly type: BillingHistoryItemType;
+
+    private readonly expirationDateTime: number;
+    private now = Math.trunc((new Date()).getTime() / 1000);
```

Comment:
why do we need additional brackets around `new Date`?

Reply:
because we cant perform getTime() on fly